runners/flink/src/main/java/org/apache/beam/runners/flink/execution/FlinkArtifactSource.java

```diff
@@ -14,12 +14,15 @@
 import org.apache.beam.runners.flink.FlinkCachedArtifactPaths;
 import org.apache.beam.runners.fnexecution.artifact.ArtifactSource;
 import org.apache.flink.api.common.cache.DistributedCache;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * An {@link org.apache.beam.runners.fnexecution.artifact.ArtifactSource} that draws artifacts
  * from the Flink Distributed File Cache {@link org.apache.flink.api.common.cache.DistributedCache}.
  */
 public class FlinkArtifactSource implements ArtifactSource {
+  private static final Logger LOG = LoggerFactory.getLogger(FlinkArtifactSource.class);
```

Comment:
I realize this is done inconsistently throughout the codebase, but could you make `log` lowercase? Loggers are not real constants since they're used for their side effects. From what I can tell, new code has been using lowercase.

Reply:
OK, looks like it's most often upper case, so we're better off keeping it here.